packages/jaeger-ui/src/types/config.tsx

```diff
@@ -164,4 +164,14 @@ export type Config = {
     menuEnabled?: boolean;
     menuLabel?: string;
   };
+
+  // traceGraph controls the trace graph under trace page
+  traceGraph?: {
```

Comment:
please move this above L157, I don't think this is "experimental"

Reply:
Moved.

I originally put it below L157 because I thought it was "undocumented" in https://www.jaegertracing.io/docs/1.42/frontend-ui/